src/commands/periscope/helpers/periscopehelper.ts

```diff
@@ -109,51 +115,67 @@ export async function getStorageInfo(
     }
 }
 
+async function extractContainerName(clusterKubeConfig: string): Promise<string | undefined> {
+        const kubectl = await k8s.extension.kubectl.v1;
+
+        if (!kubectl.available) {
+            vscode.window.showWarningMessage(`Kubectl is unavailable.`);
+            return;
+        }
+
+        // Run cluster-info to get DNS Core hostname.
+        const runCommandResult = await tmpfile.withOptionalTempFile<k8s.KubectlV1.ShellResult | undefined>(
+            clusterKubeConfig, "YAML",
+            (f) => kubectl.api.invokeCommand(`cluster-info --kubeconfig="${f}"`));
```

Comment:
Formatting suggestion: if you format this as

```
const runCommandResult = await tmpfile.withOptionalTempFile(
  clusterKubeConfig, "YAML",
  (f) => invoke(...)
);  // This closing bracket is what has moved
```

then it gives the visual appearance of nesting a la braces, and helps the reader see the scope of the callback/with-function.